src/window.rs

```diff
@@ -1394,6 +1394,22 @@ impl Window {
             .maybe_wait_on_main(|w| w.drag_resize_window(direction))
     }
 
+    /// Show [window menu] at a specified position or the current cursor position.
+    ///
+    /// This is the context menu that is normally shown when interacting with
+    /// the title bar. This is useful when implementing custom decorations.
+    ///
+    /// If `position` is `None` the current cursor position will be used.
```

Comment:
I don't like this, `the current cursor position` will be used, winit doesn't track it, and tracking it in the `Window` on some platforms, is not really fun thing to do. I'd just suggest to use `Position` since the clients must track the position already to actually handle the button clicks.